scripts/nolints.txt

```diff
@@ -7,10 +7,6 @@ apply_nolint category_theory.Kleisli.comp_def unused_arguments
 apply_nolint category_theory.Kleisli.id_def unused_arguments
 apply_nolint category_theory.Kleisli.mk doc_blame has_inhabited_instance
 
--- category_theory/endomorphism.lean
```

Comment:
Please do not manually change `nolints.txt`

Reply:
Whoops, merging master fixed this.